### DISCUSSION TITLE
Missing information about Xtext 2.28.1

### DISCUSSION THREAD

**Discussion by mmosconi1** [eclipse-xtext/xtext#2821] [Q&A] [answered]
I just stumbled upon the Xtext release 2.28.1, but can't find any information about it.
What is the difference to 2.28.0?

We are currently migrating a big DSL family to 2.28.0 and would like to now if we could/should use 2.28.1 and what implications that may have.

Thanks in advance!

**Comment by cdietrich** [comment, marked as answer]
Content is only Backport incremental maven build ( see 2.29 rel notes )
